docs/topics/object-declarations.md

```diff
@@ -3,9 +3,22 @@
 Sometimes you need to create an object that is a slight modification of some class, without explicitly declaring a new
 subclass for it. Kotlin can handle this with _object expressions_ and _object declarations_.
```

Comment:
Given that the last two bulletpoints in your list are more widely used (see the comment below), it'd make sense to rewrite the intro to highlight the more popular use cases

Reply:
good point - I rephrased it 👍